src/components/lyne-cta-button/lyne-cta-button.scss

```diff
@@ -124,17 +124,16 @@
 .button {
   display: flex;
   align-items: center;
-  padding: .5rem 1rem;
+  padding: prem(8) prem(16);
```

Comment:
Here we should already start using our spacing design tokens, e.g., padding: prem(var(--spacing-fixed-2x)) prem(var(--spacing-fixed-4x)).

Reply:
detail. The button is nowhere close to finish. The padding of 8px 16px was a random padding i choose some time ago. I will fix all the remaining styles for the button in a separate pr.